src/mono/mono/mini/aot-compiler.c

```diff
@@ -5265,6 +5274,20 @@ MONO_RESTORE_WARNING
 		}
 	}
 
+	if (acfg->aot_opts.export_symbols_outfile && export_symbols->len) {
+		char *export_symbols_out = g_string_free (export_symbols, FALSE);
+		acfg->export_symbols_outfile = fopen (acfg->aot_opts.export_symbols_outfile, "w");
```

Comment:
Is there a need to keep the file descriptor on the acfg here? I think it should just be a local and removed form the acfg, since its only opened, written to and closed in this function.

Reply:
Thanks, good point, it didn't occur to me why `data_outfile` had the FILE descriptor, but now it makes sense since its being written to in multiple places.